spec/rubocop/cop/lint/unneeded_require_statement_spec.rb

```diff
@@ -0,0 +1,48 @@
+# frozen_string_literal: true
+
+describe RuboCop::Cop::Lint::UnneededRequireStatement do
+  let(:config) { RuboCop::Config.new }
+  subject(:cop) { described_class.new(config) }
+
+  if RUBY_VERSION < '2.2'
+    context 'target ruby version < 2.2', :ruby21 do
+      it "does not registers an offense when using `require 'enumerator'`" do
+        expect_no_offenses(<<-RUBY.strip_indent)
+          require 'enumerator'
+        RUBY
+      end
+
+      it 'does not autocorrects remove unnecessary require statement' do
```

Comment:
is this test case needed? I guess `expect_no_offenses` is enough to check this cop does not auto-correct.